packages/remark-mdx/test/import-export.test.js

```diff
@@ -0,0 +1,10 @@
+const extract = require('../extract-imports-and-exports')
+const fixtures = require('./fixtures/import-export')
+
+fixtures.forEach(fixture => {
+  it(fixture.description, () => {
+    const result = extract(fixture.mdx)
+
+    expect(result).toMatchSnapshot(fixture.result)
+  })
+})
```

Comment:
Should we have a test here to make sure positional info for the nodes is correct?